bin/live/pycbc_live_combine_single_fits

```diff
@@ -149,22 +157,27 @@ for ifo in args.ifos:
         c = c[valid_alpha]
         invalphan = c / a
         mean_alpha = c.mean() / invalphan.mean()
-        q05_alpha = np.quantile(a, 0.05)
-        q05_alphas_out[ifo][counter] = q05_alpha
+        cons_alpha = np.quantile(a, qlower)
+        cons_alphas_out[ifo][counter] = cons_alpha
         alphas_out[ifo][counter] = mean_alpha
-        q95_count = np.quantile(c, 0.95)
-        q95_counts_out[ifo][counter] = q95_count * len(c)
+        cons_count = np.quantile(c, qupper)
+        cons_counts_out[ifo][counter] = cons_count * len(c)
         counts_out[ifo][counter] = c.sum()
+
+        fout[ifo + '/daily_fits/bin_%d/fit_coeff' % counter] = a
+        fout[ifo + '/daily_fits/bin_%d/counts' % counter] = c
+        fout[ifo + '/daily_fits/bin_%d/date' % counter] = ad[valid_alpha]
```

Comment:
```suggestion
        fout[f'{ifo}/daily_fits/bin_{counter:d}/fit_coeff'] = a
        fout[f'{ifo}/daily_fits/bin_{counter:d}/counts'] = c
        fout[f'{ifo}/daily_fits/bin_{counter:d}/date'] = ad[valid_alpha]
```

Reply:
not relevant given below suggestion, but I have updated to use fstrings elsewhere